karaf/features/src/main/feature/feature.xml

```diff
@@ -43,6 +43,10 @@
         <feature>package</feature>
         <feature>service</feature>
         <feature>system</feature>
+
+        <!-- org.apache.sshd.core/0.14.0 refreshes otherwise, when bcprov-ext and/or bcpkiv 1.51.0 is added -->
+        <bundle dependency="true">mvn:org.bouncycastle/bcprov-ext-jdk15on/${bouncycastle.version}</bundle>
+        <bundle dependency="true">mvn:org.bouncycastle/bcpkix-jdk15on/${bouncycastle.version}</bundle>
```

Comment:
Do we need both of these?
Can you also remove [this config](https://github.com/apache/brooklyn-dist/blob/master/karaf/apache-brooklyn/src/main/resources/etc/system.properties#L135-L142) as it's no longer needed with the new bouncycastle version.